docs/wmco-prerequisites.md

```diff
@@ -22,7 +22,7 @@ these errors, only use the appropriate version according to the cloud provider i
 | -----------         | -----------                                                             |
```

Comment:
Should it be removed from the [vSphere Windows VM Golden Image docs ](https://github.com/openshift/windows-machine-config-operator/blob/8bfc12ffe6326b977e36ff403e823ee95e317e73/docs/vsphere-golden-image.md#1-select-a-compatible-windows-server-version) too?

Reply:
As we discussed earlier, this should be addressed when we move to 20H2 or 2022 via WINC-611

Reply:
Given this, when would [Creating the vSphere Windows VM golden image](https://docs.openshift.com/container-platform/4.9/windows_containers/creating_windows_machinesets/creating-windows-machineset-vsphere.html#creating-the-vsphere-windows-vm-golden-image_creating-windows-machineset-vsphere) in the OpenShift docs get updated? I suggest that @codyhoag updates that without waiting on WINC-611.

Reply:
Yes, agree. Customer-facing documents should be up to date along with current change pre requests doc.
We can pull in WINC-611 in the upcoming sprint. But if we are moving to 2022, instead of trying 20H2 we shall directly go with 2022 golden image update.
cc: @jrvaldes

Reply:
> Given this, when would [Creating the vSphere Windows VM golden image](https://docs.openshift.com/container-platform/4.9/windows_containers/creating_windows_machinesets/creating-windows-machineset-vsphere.html#creating-the-vsphere-windows-vm-golden-image_creating-windows-machineset-vsphere) in the OpenShift docs get updated? I suggest that @codyhoag updates that without waiting on WINC-611.

Agree, and is in the [PR](https://github.com/openshift/openshift-docs/pull/40968/commits/40155fe3affae11fe2e20de3d7bda865935a655d#diff-a9972807dbbd53e5423f681de126b7ec7a6e26d87bea6a6232b1e8d4e192a686R21) already.

So, why not update it here too? [Creating the vSphere Windows VM golden image](https://docs.openshift.com/container-platform/4.9/windows_containers/creating_windows_machinesets/creating-windows-machineset-vsphere.html#creating-the-vsphere-windows-vm-golden-image_creating-windows-machineset-vsphere) should be in sync with OpenShift docs.

Reply:
> Yes, agree. Customer-facing documents should be up to date along with current change pre requests doc. We can pull in WINC-611 in the upcoming sprint. But if we are moving to 2022, instead of trying 20H2 we shall directly go with 2022 golden image update. cc: @jrvaldes

[Creating the vSphere Windows VM golden image](https://docs.openshift.com/container-platform/4.9/windows_containers/creating_windows_machinesets/creating-windows-machineset-vsphere.html#creating-the-vsphere-windows-vm-golden-image_creating-windows-machineset-vsphere) is customer-facing documentation. 

[WINC-611](https://issues.redhat.com/browse/WINC-611) and [WINC-685](https://issues.redhat.com/browse/WINC-685), for 20H2 and 2022 respectively, both account for [vsphere_ci](https://github.com/openshift/windows-machine-config-operator/tree/master/docs/vsphere_ci)